cli/util.go

```diff
@@ -86,3 +88,16 @@ func labelsToString(labels map[string]string) string {
 	}
 	return strings.Join(labelsString, ", ")
 }
+
+func actionToString(action *hcloud.Action) string {
```

Comment:
Having `actionToString` return a string of the form “Waiting for …” does not make sense to me; especially not when considering how `labelsToString` and the other similar functions work.

I would inline the `switch` to `WaitForActions`.